src/ScrumBoardItBundle/Entity/Issue/AbstractIssue.php

```diff
@@ -432,6 +411,30 @@ public function setTimeBox($timeBox)
     }
 
     /**
+     * Return On Investment getter.
+     *
+     * @return number
+     */
+    public function getReturnOnInvestment()
+    {
+        return $this->returnOnInvestment;
+    }
+
+    /**
+     * Return On Investment setter.
+     *
+     * @return self
+     */
+    public function setReturnOnInvestment()
+    {
+        if (!empty($this->businessValue) && !empty($this->complexity) && $this->businessValue > 0 && $this->complexity > 0) {
```

Comment:
Round 0 plz, there is no need to decimal.
You should test if BV >= CT too.
Futur user could have specific ladder for businessValue, so it could be possible to have a businessValue < complexity.

You might